omnibus/files/private-chef-cookbooks/private-chef/templates/default/nginx/nginx_chef_api_lb.conf.erb

```diff
@@ -71,9 +83,11 @@
     #   /compliance/organizations/ORG/owners/OWNER/compliance[/PROFILE]
     location ~ (?:/compliance)?/organizations/([^/]+)/owners/([^/]+)/compliance(.*) {
```

Comment:
It sounds like we're going to match something like "^/compliance" and ignore the details going forward; let's make sure we '^' anchor the new regular expression.